src/wp-includes/class-wp-theme-json.php

```diff
@@ -2963,9 +2963,9 @@ public function get_root_layout_rules( $selector, $block_metadata ) {
 			// Alignfull children of the container with left and right padding have negative margins so they can still be full width.
 			$css .= '.has-global-padding > .alignfull { margin-right: calc(var(--wp--style--root--padding-right) * -1); margin-left: calc(var(--wp--style--root--padding-left) * -1); }';
 			// Nested children of the container with left and right padding that are not wide or full aligned do not get padding, unless they are direct children of an alignfull flow container.
```

Comment:
```suggestion
			// Nested children of the container with left and right padding that are not full aligned do not get padding, unless they are direct children of an alignfull flow container.
```

Reply:
Oh well spotted! Should update the comment on the Gutenberg side too.

Reply:
The changes already their in GB side.